routes/router.js

```diff
@@ -661,13 +660,24 @@ router.route('/uploadFile')
                                 })
                                 // remember to set the right name for the database
                                 neume['imagesBinary'] = image_binaries;
+                                continue;
 
                             }
+                            // if the cell is empty then continue
+                            else if (!text_elements) {
+                                continue;
```

Comment:
Are these continues necessary? It appears that the loop ends with this if/else chain and the continue statement is unnecessary.